app/src/main/kotlin/com/github/premnirmal/ticker/settings/WidgetSettingsFragment.kt

```diff
@@ -128,6 +130,21 @@ class WidgetSettingsFragment : BaseFragment(), OnClickListener {
               InAppMessage.showMessage(activity!!, R.string.layout_updated_message)
             })
       }
+
+      R.id.setting_widget_width -> {
+        showDialogPreference(R.array.widget_width_types,
+                DialogInterface.OnClickListener { dialog, which ->
+                  widgetData.setWidgetSizePref(which)
+                  setWidgetSizeSetting(widgetData)
+                  dialog.dismiss()
+                  broadcastUpdateWidget()
+                  /*if (which == 2) {
+                    showDialog(getString(R.string.change_instructions))
+                  }*/
+                  InAppMessage.showMessage(activity!!, R.string.widget_width_updated_message)
```

Comment:
You can use `requireActivity()` instead of `activity!!`

Reply:
Just used same way you developed in all other functions in same class such as setting_layout_type